### PR TITLE
Action is coming

### PR DESCRIPTION
**Pull request by LevFlavien**
g pa testé mdr

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -404,7 +404,7 @@ public function fast($sql, $opt = null, $i = null, $mode = PDO::FETCH_BOTH){
                     }
                 }
             }
-            
+
```
Comment (Adrriii) [inline index.php:407]: keep my white lines plz

File: index.php [hunk 2]
```diff
@@ -413,7 +413,8 @@ public function fast($sql, $opt = null, $i = null, $mode = PDO::FETCH_BOTH){
         <?php
     }
     ?>
-    <img class="satourn" class="satourn" src="https://cdn.drawception.com/images/panels/2018/2-13/AypBaBreLw-2.png"/>
+    <a href="https://osu.ppy.sh/s/977678"><img class="satourn" class="satourn" src="https://cdn.drawception.com/images/panels/2018/2-13/AypBaBreLw-2.png"/></a>
+    <a href="https://osu.ppy.sh/s/1010740"><img class="satourn" class="satourn" src="https://b.ppy.sh/thumb/1010740l.jpg?update=2019-07-28%2018:11:32"/></a>
```
Comment (Adrriii) [inline index.php]: class before href, always. and src before href, which is after class

File: index.php [hunk 3]
```diff
@@ -413,8 +413,15 @@ public function fast($sql, $opt = null, $i = null, $mode = PDO::FETCH_BOTH){
...
+    <div style="display:flex" display="💪">
+        <a href="https://osu.ppy.sh/s/960544"><img class="satourn" class="satourn" src="http://cdn.shopify.com/s/files/1/1061/1924/products/Poop_Emoji_7b204f05-eec6-4496-91b1-351acc03d2c7_grande.png?v=1480481059"/></a>
+        <a href="CUNU OMGFDOMG PACK 2 WHEN"><img class="satourn" class="satourn" src="PUTAIN"/></a>
+        <a href="https://osu.ppy.sh/s/977678"><img class="satourn" class="satourn" src="https://cdn.drawception.com/images/panels/2018/2-13/AypBaBreLw-2.png"/></a>
+        <a href="https://osu.ppy.sh/s/991882"><img class="satourn" class="satourn" src="no_image_edition" title="tamer" alt="no image edition"/></a>
+        <a href="https://osu.ppy.sh/s/999478"><img class="satourn" class="satourn" src="https://i.ytimg.com/vi/5QvgLlFyeok/hqdefault.jpg"/></a>
+        <a href="https://osu.ppy.sh/s/1010740"><img class="satourn" class="satourn" src="https://b.ppy.sh/thumb/1010740l.jpg?update=2019-07-28%2018:11:32"/></a>
+        <a href="https://osu.ppy.sh/s/1014492"><img class="satourn" class="satourn" src="https://www.sciencesetavenir.fr/assets/img/2016/06/23/cover-r4x3w1000-5c4095e068d41-pieds.jpg2"/></a>
+    </div>
```
Comment (Adrriii) [inline index.php:424]: ok

**Comment by LevFlavien** [comment]
https://www.youtube.com/playlist?list=PLVAdhvP24j5GG7mxkWNswdClgbidsQKy0

File: .gitignored [hunk 4]
```diff
@@ -0,0 +1 @@
+fuck adri
```
Comment (Adrriii) [inline .gitignored:1]: capitalization

**Review by Adrriii — APPROVED** [review]

File: index.php (hunk 1 above)
Comment (LevFlavien) [reply index.php:407]: c phptempete

**Comment by LevFlavien** [comment]
mersi bien lami

**Pull request merged**